tests/kernels/test_cutlass.py

```diff
@@ -11,6 +11,22 @@
 from vllm import _custom_ops as ops
 from vllm.platforms import current_platform
 
+MNK_FACTORS = [
+    (1, 256, 128),
+    (1, 16384, 1024),
+    (1, 24576, 496),
+    (16, 256, 496),
+    (16, 16384, 128),
+    (33, 1024, 1024),
+    (33, 8192, 128),
+    (64, 2048, 496),
+    (64, 16384, 1024),
+    (100, 8192, 496),
+    (512, 256, 1024),
+    (512, 16384, 128),
+    (512, 24576, 128),
+]
```

Comment:
can we also add batch sizes 32 and 128 and 256 please. something like ,
(32, 4096, 4096)
(32, 8192, 4096)
(128, 4096, 4096)
(256, 4096, 4096)

also for 89 we have a variety of kernels based on M and N. Can we add these kernels for sm89 alone ? 
K = any (for e.g. 4096)
sm89 fp8
```
(512, 4096, K)
(512, 8192, K)
(512, 16384, K)
(256, 4096, K)
(256, 8192, K)
(128, 8192, K)
(128, 16384, K)
(128, 32768, K)
(64, 8192, K)
(64, 16384, K)
(64, 32768, K)
(32, 8192, K)
(32, 16384, K)
(32, 32768, K)
(16, 8192, K)
(16, 24576, K)
(16, 32768, K)
```

sm89 int8
```
(512, 4096, K)
(512, 8192, K)
(512, 16384, K)
(512, 32768, K)
(256, 4096, K)
(256, 8192, K)
(256, 16384, K)
(256, 32768, K)
(128, 8192, K)
(128, 16384, K)
(128, 32768, K)
(64, 8129, K)
(64, 16384, K)
(32, 8129, K)
(32, 16384, K)
(16, 8129, K)
(16, 16384, K)
```

Reply:
Why do we need to test all of the in-between values? Since this isn't a performance test, I would think we just want to validate the extents or edge cases of the range of acceptable values rather than the intermediates

Reply:
by `in-between values` I believe you mean the Ns . For sm_89's I took a cursory look at, 
 - https://github.com/vllm-project/vllm/blob/main/csrc/quantization/cutlass_w8a8/scaled_mm_c2x_sm89_fp8_dispatch.cuh and 
 - https://github.com/vllm-project/vllm/blob/main/csrc/quantization/cutlass_w8a8/scaled_mm_c2x_sm89_fp8_dispatch.cuh
 for coming up with the gemm shapes. There, we dispatch different kernels for different Ns. 

I believe we should test all the available at least once. 

 However, I dont think the list I shared is the mininum set - l'll take another look to see if I can come up with a shorter list.

Reply:
>However, I dont think the list I shared is the mininum set - l'll take another look to see if I can come up with a shorter list.
@mgoin - To the shapes you already have, can you add the following shapes please,
```
(512, 8192, 4096)
(256, 4096, 4096)
(128, 32768, 4096)
(32, 8192, 4096)
(32, 16384, 4096)
(16, 24576, 4096)
```
This list of shapes should invoke all the cutlass kernels we have.

Reply:
Okay that is certainly more minimal, thanks and I'll put these in!